pkg/container/container.go

```diff
@@ -285,7 +288,7 @@ func (c *container) SetConfig(config *types.ContainerConfig, sboxConfig *types.P
 		return errors.New("sandbox config is already set")
 	}
 
-	c.config = config
+	c.config = cfg
```

Comment:
Nit: these hunks does not seem to be relevant.

Reply:
we now import our config package, and we shadow the import unless we change the variable name